src/Orleans.TestingHost/Utils/TestingUtils.cs

```diff
@@ -40,8 +41,9 @@ public static string CreateTraceFileName(string nodeName, string clusterId)
             {
                 Directory.CreateDirectory(traceFileFolder);
             }
-
-            var traceFileName = $"{traceFileFolder}\\{clusterId}_{nodeName}.log";
+            DateTime now = DateTime.UtcNow;
+            string DateTimeFormat = @"yyyy-MM-dd\tHH-mm-ss-fff";
+            var traceFileName = $"{traceFileFolder}\\{clusterId}_{nodeName}_{now.ToString(DateTimeFormat, CultureInfo.InvariantCulture)}.log";
```

Comment:
Should use Path.Combine instead of explicit `\`.

Maybe add `Z` to the end to signify that it's a UTC time

Reply:
`DateTime.UtcNow.ToString("o")` should be better...

Reply:
This is a utility for the TestCluster (which BTW we are updating in the TestCluster revamp branch if I'm not mistaken), and for these, clusterId already includes the timestamp in the name... This would have the timestamp 2 times in our tests, correct?